src/components/Character.js

```diff
@@ -1,7 +1,24 @@
-import React from "react";
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
 
-const Character = props => {
-  return <li>{props.character.name}</li>;
+const Character = ({ character }) => {
+	return <CharacterStyled>{character.name}</CharacterStyled>;
 };
 
 export default Character;
+
+Character.propTypes = {
+	character: PropTypes.object.isRequired
+};
+
+const CharacterStyled = styled.li`
```

Comment:
You put the styling with styled-components below your component logic. Awesome! Way easier for me and for you to read :)